<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Псевдоним. Возвращение</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>game-console.png</normaloff>game-console.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="text1">
        <property name="text">
         <string>Задать количество камней</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="pebbleAmount">
        <property name="minimum">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="setButton">
        <property name="text">
         <string>Задать</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLCDNumber" name="remainingAmount"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="text2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Сколько камней взять?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="takeAmount">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>3</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="takeButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Взять</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="matchHistory">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>125</height>
       </size>
      </property>
      <property name="documentTitle">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Здесь будет история  игры.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="winLabel">
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
